### ISSUE TITLE
Config option to set test folder path / pattern

### ISSUE THREAD

**Issue by ghost** [compulim/vscode-mocha#1]
- I miss a option to set my own test path/pattern, something like that:

``` json
{ 
    "mocha.options": {
        "watch": "scripts/components/**/*.js",
        "runs":  "__tests__/**/*.spec.js"
    }
}
```

Because I work on different projects and I can't change 
- a) the route/naming of test folder and 
- b) the naming/suffixing of test files.

So, it would be very nice, if I have options to config something.

**Comment by compulim** [comment]
Will do.

**Comment by compulim** [comment]
Fixed in commit a0193f1d37d7496d071c1bd5c1deab00b45411fc.

Published to marketplace with [version 0.1.1](https://marketplace.visualstudio.com/items?itemName=Compulim.vscode-mocha).

We don't have any watch loop right now. So I am just adding the test files glob.

You can configure it with settings:

```
// Mocha: Glob to search for test files
"mocha.files.glob": "test/**/*.js",

// Mocha: Globs to ignore when searching for test files
"mocha.files.ignore": [
  "**/.git/**/*",
  "**/node_modules/**/*"
],
```

Please let me know if you need more help on fitting this extension to your project.

If you like this extension, don't forget to [star](https://github.com/compulim/vscode-mocha/stargazers) this repository and [rate us](https://marketplace.visualstudio.com/items?itemName=Compulim.vscode-mocha) in marketplace.
